### PR TITLE
LeetCode problem # 1232, 'Check If It Is a Straight Line'

### PR DESCRIPTION
**Pull request by simona1**
JavaScript solution

### CODE REVIEW

File: leetcode/check-straight-line-solution.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/**
+ * LeetCode problem # 1232, 'Check If It Is a Straight Line'
+ * https://leetcode.com/problems/check-if-it-is-a-straight-line
+ */
+
+function isStraight([[ax, ay], [bx, by], [cx, cy]]) {
+  return (by - ay) * (cx - bx) === (cy - by) * (bx - ax);
```
Comment (Judah-Silva) [inline leetcode/check-straight-line-solution.js:12]: Maybe add a comment to make this easier to read and understand

File: leetcode/check-straight-line-solution.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+function checkStraightLine(coordinates) {
+  if (coordinates.length === 2) {
```
Comment (Judah-Silva) [inline leetcode/check-straight-line-solution.js]: The if statement is not necessary

File: leetcode/check-straight-line-solution.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/**
+ * LeetCode problem # 1232, 'Check If It Is a Straight Line'
+ * https://leetcode.com/problems/check-if-it-is-a-straight-line
+ */
+
+function isStraight([[ax, ay], [bx, by], [cx, cy]]) {
```
Comment (Judah-Silva) [inline leetcode/check-straight-line-solution.js]: No need to include arguments in an array

**Review by Judah-Silva — APPROVED** [review]

File: leetcode/check-straight-line-solution.js (hunk 2 above)
Comment (simona1) [reply leetcode/check-straight-line-solution.js]: Removed

File: leetcode/check-straight-line-solution.js (hunk 1 above)
Comment (simona1) [reply leetcode/check-straight-line-solution.js:12]: Done!

**Pull request merged**